js/load-scene.js

```diff
@@ -63,6 +63,7 @@ AFRAME.registerComponent('loadscene', {
 
 
     document.querySelector('a-scene').addEventListener('loaded', function () {
+      document.getElementById('get-ready').style.display = 'none';
       document.getElementById('intro-audio').play();
```

Comment:
_(not sure, I may have screwed up and posted my comments in the wrong spot, my bad.)_

can you change this to happen when the Enter VR button is pressed? that is, call these two lines directly in the `click` event handler for the Enter VR button.

since Firefox 63, [Firefox's `autoplay` policy](https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide#Autoplay_availability) disallows audio from being unless the `play()` call happens in a user-initiated event. that must be a `click`', `touch*`, or `key*` event (`mouse*` events do not).

(as you know, Chrome and Safari shipped their autoplay policies many versions back.) 

here are the warnings and errors I see in Firefox (but they apply to all modern browsers now):

`Autoplay is only allowed when approved by the user, the site is activated by the user, or media is muted.`

`NotAllowedError: The play method is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.`

<hr>

# Chrome

## `master`
2 errors:
![image](https://user-images.githubusercontent.com/203725/52021540-6b9cf700-24aa-11e9-9d46-8821938feda3.png)

## this branch
4 errors:
![image](https://user-images.githubusercontent.com/203725/52021562-8cfde300-24aa-11e9-85f1-43903b87b65a.png)

<hr>

# Firefox

## `master`
![image](https://user-images.githubusercontent.com/203725/52021620-bd458180-24aa-11e9-8cc5-8dd4861e69fe.png)

## this branch
![image](https://user-images.githubusercontent.com/203725/52021603-b6b70a00-24aa-11e9-88df-16833cfb813b.png)

Reply:
I'll deal with this in another PR.

Reply:
filed issue #87 to handle these two errors:

- `NotAllowedError: The play method is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.`
- `TypeError: Argument 1 of BaseAudioContext.decodeAudioData does not implement interface ArrayBuffer.`